srcpkgs/xdgmenumaker/template

```diff
@@ -1,16 +1,16 @@
 # Template file for 'xdgmenumaker'
 pkgname=xdgmenumaker
-version=1.6
-revision=2
+version=2.0
+revision=1
 build_style=gnu-makefile
 hostmakedepends="txt2tags"
 depends="python3-xdg python3-gobject gobject-introspection python3-Pillow"
```

Comment:
```suggestion
depends="python3-xdg python3-gobject gobject-introspection python3-Pillow gtk+3"
```

Reply:
It is not required dependency for build, the software does not use gtk+3. What is the reason for using gtk+3?

Reply:
`$depends` is a list of runtime dependencies, not build dependencies. xdgmenumaker uses it [here](https://github.com/gapan/xdgmenumaker/blob/7dd8a63abd0de0151708339532c6a717b2741328/src/xdgmenumaker#L311) to figure out the icon path.

Reply:
Done!